app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+
+    has_many :events, foreign_key: "creator_id", class_name: "Event"
+    has_many :attended_events, foreign_key: "attendee_id"
```

Comment:
I am very confused about this. Does `attended_events` mean `attendances` ? Because right now as it is named, I was thinking this was just an alias for the `events`. Let's rename this to something more intuitive.